06-building-a-program-start-to-finish/writing-tests/output/invertedIndexSpec.js

```diff
@@ -0,0 +1,263 @@
+const books = require('../assets/books.json');
+const wrongData = require('../assets/wrongData.json');
+const emptyFile = require('../assets/emptyFile.json');
+const smallFile = require('../assets/smallFile.json');
+
+describe('InvertedIndex Class', () => {
+  beforeAll(() => {
+    this.invertedIndex = new InvertedIndex();
+  });
+
+  describe('Class instantaion', () => {
+    it('should be instantiated with the "new" keyword', () => {
+      const init = () => {
+        InvertedIndex();
+      };
+      expect(init)
+        .toThrowError(`Class constructor` +
```

Comment:
Thank you for pointing this out, however this is an outdated file i believe as my current branch stored the error message in a variable, but i have also taken your advice and stored it in a JSON object, thereby allowing me to require it and use it when needed.